.circleci/config.yml

```diff
@@ -1,46 +1,34 @@
 version: 2.1
 orbs:
-  singularity: singularity/singularity@1.0.10
   python: circleci/python@0.3
 jobs:
-  singularity-image:
-    executor: singularity/machine
+  docker:
+    machine: true
     steps:
       - checkout
-      - singularity/install-go:
-          go-version: '1.13'
-      - singularity/debian-install-3:
-          singularity-version: 3.5.0
       - run:
-          # copied from singularity/build-image to add branch-specific build
-          name: Build Singularity Container
+          name: Build docker image
           command: |
-            NOSUDO=false
-            which sudo || NOSUDO=true
-            if [[ "$NOSUDO" == "false" ]]; then
-                if [[ $EUID == 0 ]]; then export SUDO=""; else export SUDO="sudo"; fi
-            fi
-            if [ -f "$BASH_ENV" ]; then source $BASH_ENV; fi
-            $SUDO SINGULARITYENV_OPHYS_ETL_TAG=${CIRCLE_BRANCH} singularity build suite2p_container.img src/ophys_etl/transforms/suite2p_container.def
+            docker build --build-arg OPHYS_ETL_TAG=${CIRCLE_BRANCH} -t alleninstitutepika/ophys_etl_pipelines:${CIRCLE_SHA1} docker/
```

Comment:
In ophys_nway, I found it convenient to pass the actual commit has into the container build:
https://github.com/AllenInstitute/ophys_nway_matching/blob/65f5c3d28c567e9f5cb31c5fe84b8f952a889a7b/.circleci/config.yml#L48-L51
so that it was unambiguous which commit is being used. I could then make the module aware ofd that env variable, so that it could appear in logs:
```
INFO:__main__:NWAY_COMMIT_SHA 65f5c3d28c567e9f5cb31c5fe84b8f952a889a7b
INFO:__main__:Nway matching version 0.4.1
```
what do you think about this for `ophys_etl`?

Reply:
Sure, I can go ahead and add this to the dockerfile.